assignment_4.rb

```diff
@@ -0,0 +1,11 @@
+def solution(day, k)
+  days = %w[Mon Tue Wed Thu Fri Sat Sun]
```

Comment:
Maybe move to outside function like this
```suggestion
DAYS = %w[Mon Tue Wed Thu Fri Sat Sun]

def solution(day, k)
```